### ISSUE TITLE
1.19: is there a way to disbale and enable party chat spy?

### ISSUE THREAD

**Issue by LilShadow19** [kangarko/ChatControl#2320]
### Are you using MySQL?

No

### Are you using BungeeCord/Velocity?

No

### Question

For chat spy is there a command to disbale and enable party chat, like /spy toggle private_message?

**Comment by kangarko** [comment]
I am sorry, I am not sure if I understand you here, can you rephrase please? You can disable spying on channels at once I think

**Comment by ElBananaa** [comment]
Closing this due to innactivity.
Feel free to keep commenting down below if you need anything else (we will still receive a notification)
